.github/workflows/release-it.yml

```diff
@@ -0,0 +1,37 @@
+# #########################################################################
+# Creates a new release using `workflow_dispatch` event trigger with `type`
+# as input to describe the type of release to create
+name: 'Release-it: Create a new release on demand'
+
+on:
+  workflow_dispatch:
+    inputs:
+      type:
+        description: 'Type. Can be `patch` `minor` or `major`'
+        required: true
+        default: 'patch'
+jobs:
+  release-it:
+    if: contains('["robertsLando","chrisns"]', github.actor)
```

Comment:
I'm not sure if you even need that strange array syntax - it is a string after all ;)

Reply:
Dunno how I missed that but good catch! :) Fixed